It looks like some clang compile error, take a look at:


bazel-out/darwin_x86_64-opt/bin/tensorflow/core/kernels/_objs/parse_tensor_test/tensorflow/core/kernels/parse_tensor_test.o): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
tensorflow/core/kernels/parse_tensor_test.cc:173:25: error: non-constant-expression cannot be narrowed from type 'double' to 'float' in initializer list [-Wc++11-narrowing]
      return complex64{ x / 8., x / 2. };
                        ^~~~~~
tensorflow/core/kernels/parse_tensor_test.cc:173:25: note: insert an explicit cast to silence this issue
      return complex64{ x / 8., x / 2. };
                        ^~~~~~
                        static_cast<float>( )
tensorflow/core/kernels/parse_tensor_test.cc:173:33: error: non-constant-expression cannot be narrowed from type 'double' to 'float' in initializer list [-Wc++11-narrowing]
      return complex64{ x / 8., x / 2. };
                                ^~~~~~
tensorflow/core/kernels/parse_tensor_test.cc:173:33: note: insert an explicit cast to silence this issue
      return complex64{ x / 8., x / 2. };
                                ^~~~~~
                                static_cast<float>( )

--- a/tensorflow/core/kernels/parse_tensor_test.cc
+++ b/tensorflow/core/kernels/parse_tensor_test.cc
@@ -170,7 +170,8 @@ TEST_F(SerializeTensorOpTest, SerializeTensorOpTest_uint8) {
 
 TEST_F(SerializeTensorOpTest, SerializeTensorOpTest_complex64) {
   MakeOp<complex64>(TensorShape({}), [](int x) -> complex64 {
-      return complex64{ x / 8., x / 2. };
+      return complex64{ static_cast<float>(x / 8.),
+            static_cast<float>(x / 2.) };
     });
   TF_ASSERT_OK(RunOpKernel());
   Tensor parse_output;

